gulpfile.js

```diff
@@ -184,7 +184,7 @@ gulp.task( 'browser-sync', function() {
     .pipe( sourcemaps.init( { loadMaps: true } ) )
     .pipe( autoprefixer( AUTOPREFIXER_BROWSERS ) )
 
-    .pipe( sourcemaps.write ( styleDestination ) )
+//    .pipe( sourcemaps.write ( styleDestination ) )
```

Comment:
I can see that now. But to address it, we need to see if the source maps can get correct root path. After commenting this line, you must not have any source maps. Am I right?